llm_on_ray/inference/predictor_deployment.py

```diff
@@ -394,8 +394,13 @@ async def __call__(self, http_request: Request) -> Union[StreamingResponse, JSON
         streaming_response = request["stream"]
         input = request["text"]
         config = request["config"]
-
+        if config.get("debug_mode", False):
+            logger.debug(f"Print config received from json: {config}")
+            logger.debug(f"Print inputs for prompts: {input}")
+        # return prompt or list of prompts preprocessed
         prompts = self.preprocess_prompts(input)
+        if config.get("debug_mode", False):
```

Comment:
debug mode should be enabled only from server command line rather than from client request.

Reply:
I believe both server and client side should have debug log for better locating bugs.

Reply:
The problem here is that the config here is from http request, service logic depends on this to print debug message, this will create a potential security holes as we can't control what client will send to the server.